app/views/profiles/edit.html.erb

```diff
@@ -1,74 +1,77 @@
-<html>
-  <head>  
-    <link href="https://fonts.googleapis.com/css?family=Libre+Franklin" rel="stylesheet">
-  </head>
```

Comment:
Good catch removing this stuff. Are there other views that have the same issue?

Reply:
As far as including fonts, yeah. We're addressing as part of #210